contracts/Cartographer.sol

```diff
@@ -104,6 +105,7 @@ contract Cartographer is Ownable, Initializable, ReentrancyGuard {
     address public expeditionTreasuryAdd;                                       // Expedition Treasury address, intermediate address to convert to stablecoins
     address public trustedSeederAdd;                                            // Address that seeds the random number generation every 2 hours
     ElevationHelper elevationHelper;
+    SummitVRFModule summitVRFModule;
```

Comment:
Changes look great, can we update the item in the state of ElevationHelper to look at the interface so that it can be replaced in the future too. After that it can be merged.

Thanks!

Reply:
I just updated summitVRFModule state variable to summitVRFModuleAdd with address format